fix(Changelog.md): Fixed broken links (#1246)
eclipse-tractusx/eclipse-tractusx.github.io: src/pages/CHANGELOG.md

| [Industry Core KIT](https://eclipse-tractusx.github.io/docs-kits/category/industry-core-kit) | n/a | [1.3.0](https://eclipse-tractusx.github.io/docs-kits/kits/industry-core-kit/changelog) |
| [Knowledge Agents](https://github.com/eclipse-tractusx/knowledge-agents) | remoting-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/remoting-agent-1.14.24)<br/>provisioning-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/provisioning-agent-1.14.24)<br/>conforming-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/conforming-agent-1.14.24)<br/>matchmaking-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/matchmaking-agent-1.14.24) | remoting-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24)<br/>provisioning-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24)<br/>conforming-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24)<br/>matchmaking-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24) |

> | [(Knowledge) Agents KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/knowledge-agents-kit) | n/a | [1.2.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/knowledge-agents/Knowledge%20Agent%20Changelog#120---2024-07-29) |

Reviewer: Hi think this is not correct. Even if @jSchuetz88  switched to the correct changelog id in #1247 , i think the kit foldername is still not knowledge-agents-kit. and in this case this link cant work.

@jSchuetz88 should we spend the time and also rename the kit.? Effort... yes ... but consistent

Reviewer 2: This comment should be resolved, we fixed the agents KIT name. Its knowledge-agents-kit now

| Component | Helm Chart (s) | App-/KIT Version (s) |
| :-------- | :-------------: | :------------------: |
| [Knowledge Agents AAS Bridge](https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge) | aas-bridge: [1.13.7](https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge/releases/tag/aas-bridge-1.13.7) | [1.13.7](https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge/releases/tag/v1.13.7) |

> | [Supply Chain Disruption Notifications Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/supply-chain-disruption-notification-kit) | n/a | [1.0.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/supply-chain-disruption-notification-kit/Supply%20Chain%20Disruption%20Notifications%20Changelog#100---2024-08-01) |

Reviewer 2: ```suggestion
| [Supply Chain Disruption Notifications Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/supply-chain-disruption-notification-kit) | n/a | [1.0.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/supply-chain-disruption-notification-kit/changelog#100---2024-08-01) |
```

I think you missed one.

| [Discovery Finder](https://github.com/eclipse-tractusx/sldt-discovery-finder) | discoveryfinder: [0.3.1](https://github.com/eclipse-tractusx/sldt-discovery-finder/releases/tag/discoveryfinder-0.3.1) | [v0.4.0](https://github.com/eclipse-tractusx/sldt-discovery-finder/releases/tag/v0.4.0) |
| [Discovery Service (BPN Finder)](https://github.com/eclipse-tractusx/sldt-bpn-discovery) | bpndiscovery: [0.3.1](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/bpndiscovery-0.3.1) | [v0.4.0](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/v0.4.0) |
| [Eclipse Data Space Connector (Tractus-X EDC)](https://github.com/eclipse-tractusx/tractusx-edc) | tractusx-connector: [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) |
| [EcoPass Kit](https://eclipse-tractusx.github.io/docs-kits/category/eco-pass-kit) | n/a | [1.5.0](https://eclipse-tractusx.github.io/docs-kits/kits/Eco_Pass_KIT/changelog) |

> | [ESS Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/environmental-and-social-standards-kit) | n/a | [0.3.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/environmental-and-social-standards-kit/ESS%20Kit%20Changelog#030---2024-07-11) |

Reviewer 2: ```suggestion
| [ESS Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/environmental-and-social-standards-kit) | n/a | [0.3.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/environmental-and-social-standards-kit/changelog#030---2024-07-11) |
```

| [Eco Pass KIT](https://eclipse-tractusx.github.io/docs-kits/category/eco-pass-kit)                                                                                |                                                                                                  n/a                                                                                                   |                                                                                                                                                           [1.4.0](https://eclipse-tractusx.github.io/docs-kits/kits/Eco_Pass_KIT/changelog)                                                                                                                                                            |

> | [Environmental and Social Standards (ESS) KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/environmental-and-social-standards-kit)                                                     |                                                                                                  n/a                                                                                                   |                                                                                                                                                        [0.2.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/environmental-and-social-standards-kit/ESS%20Kit%20Changelog#020---2024-05-23)                                                                                                                                                        |

Reviewer: looks not correct `/ESS%20Kit%20Changelog`

|:------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------:|
| [Certificate Management KIT](https://eclipse-tractusx.github.io/docs-kits/category/certificate-management-kit)                |                                        n/a                                         |                         [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/Certificate%20Management%20Kit/Changelog)                          |
| [Data Governance KIT](https://eclipse-tractusx.github.io/docs-kits/category/data-governance-kit)                              |                                        n/a                                         |                [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/Data%20Governance%20Kit/Data%20Governance%20Kit%20Changelog)                |

> | [Environmental and Social Standards (ESS) KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/environmental-and-social-standards-kit)                 |                                        n/a                                         |                               [0.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/environmental-and-social-standards-kit/ESS%20Kit%20Changelog#010---2024-03-06)                               |

Reviewer 2: ```suggestion
| [Environmental and Social Standards (ESS) KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/environmental-and-social-standards-kit)                 |                                        n/a                                         |                               [0.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/environmental-and-social-standards-kit/changelog#010---2024-03-06)                               |
```

| [Industry Core KIT](https://eclipse-tractusx.github.io/docs-kits/category/industry-core-kit)                                  |                                        n/a                                         |                  [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/industry-core-kit/changelog)                  |

> | [Manufacturing as a Service (MaaS) KIT](https://eclipse-tractusx.github.io/docs-kits/category/manufacturing-as-a-service-kit) |                                        n/a                                         | [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/manufacturing-as-a-service-kit/Manufacturing%20as%20a%20Service%20KIT%20Changelog) |

Reviewer 2: ```suggestion
| [Manufacturing as a Service (MaaS) KIT](https://eclipse-tractusx.github.io/docs-kits/category/manufacturing-as-a-service-kit) |                                        n/a                                         | [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/manufacturing-as-a-service-kit/changelog) |
```

### Unchanged, untested

| Component                                                                                                                                                         |                                                 Helm Chart (s)                                                  |                                                                      App-/KIT Version (s)                                                                       |
|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------:|

> | [Behavioral Twin: Health Indicators (HI) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/hi/overview)                                      |                                                       n/a                                                       |                   [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour-twin-kit/Changelog%20Health%20Indicator%20Kit)                   |

Reviewer 2: ```suggestion
| [Behavioral Twin: Health Indicators (HI) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/hi/overview)                                      |                                                       n/a                                                       |                   [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/behaviour-twin-kit/changelog)                   |
```

| [Behavioral Twin: Model and Data processing (MDP) KIT](hhttps://eclipse-tractusx.github.io/docs-kits/next/category/model-based-development-and-data-processing-kit) |                                                       n/a                                                       | [0.1.1](https://eclipse-tractusx.github.io/docs-kits/next/kits/model-based-development-and-data-processing-kit/changelog#011---2023-12-20) |

> | [Behavioral Twin: Remaining useful Life (RUL) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/rul/overview)                           |                                                       n/a                                                       |               [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20RuL%20Kit/Changelog%20Remaining%20Useful%20Life%20Kit)               |

Reviewer 2: ```suggestion
| [Behavioral Twin: Remaining useful Life (RUL) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/rul/overview)                           |                                                       n/a                                                       |               [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/behaviour-twin-kit/changelog)               |
```

### Using helm with central helm registry

You can search all released helm charts with the following commands:

```shell
helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
helm repo update tractusx-dev
helm search repo tractusx-dev --versions
```

For installation details, please see the referenced Repositories / Releases.

## [23.12] - 2023-12-08

### Added

| Component                                                                                                                                                         | Helm Chart (s) |                                                                          App-/KIT Version (s)                                                                          |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------:|

> | [Behavioral Twin: Health Indicators (HI) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/hi/overview)                                      |      n/a       |                   [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/behaviour-twin-kit/adoption-view%20Health%20Indicator%20Kit)                    |

Reviewer 2: ```suggestion
| [Behavioral Twin: Health Indicators (HI) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/hi/overview)                                      |      n/a       |                   [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/behaviour-twin-kit/changelog)                    |
```

| [Behavioral Twin: Model and Data processing (MDP) KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/model-based-development-and-data-processing-kit) |      n/a       | [0.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/model-based-development-and-data-processing-kit/changelog#010---2023-12-08) |
| [Circularity KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/circularity-kit/adoption-view)                                                           |      n/a       |                                     [0.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/circularity-kit/changelog)                                     |
| [Demand and Capacity Management (DCM) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/demand-and-capacity-management-kit/adoption-view/overview)                                                   |      n/a       |                                       [3.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/demand-and-capacity-management-kit/changelog)                                        |

> | [Knowledge Agent KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/knowledge-agents-kit)                                                                           |      n/a       |                                    [1.0.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/knowledge-agents/Knowledge%20Agent%20Changelog#100---2023-11-24)                                     |

Reviewer 2: ```suggestion
| [Knowledge Agent KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/knowledge-agents-kit)                                                                           |      n/a       |                                    [1.0.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/knowledge-agents/changelog#100---2023-11-24)                                     |
```

| [OSim KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/online-simulation-kit)                                                              |                                                               n/a                                                                |                    [1.0.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/online-simulation-kit/changelog#100---2023-08-18)                     |
| [Product Carbon Footprint (PCF) KIT](https://eclipse-tractusx.github.io/docs-kits/next/category/product-carbon-footprint-exchange-kit)                            |                                                               n/a                                                                |          [0.1.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/product-carbon-footprint-exchange-kit/changelog#100---2024-03-08)           |

> | [Behavioral Twin: Remaining useful Life (RUL) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/rul/overview) |                                                               n/a                                                                | [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20RuL%20Kit/Changelog%20Remaining%20Useful%20Life%20Kit) |

Reviewer: looks not correct

| Component                                                                                                                               |                                                      Helm Chart (s)                                                      |                                                           App Version (s)                                                           |
|:----------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------:|

> | [Behavioral Twin: Remaining useful Life (RUL) KIT](https://eclipse-tractusx.github.io/docs-kits/next/kits/behaviour-twin-kit/use-cases/rul/overview) |                                                           n/a                                                            | [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20RuL%20Kit/Changelog%20Remaining%20Useful%20Life%20Kit) |

Reviewer: looks not correct
